graphqlapi/src/main/java/io/stargate/graphql/schema/DmlSchemaBuilder.java

```diff
@@ -527,17 +545,53 @@ public GraphQLObjectType buildType(Table table) {
 
   private GraphQLFieldDefinition.Builder buildOutputField(Table table, Column columnMetadata) {
     return new GraphQLFieldDefinition.Builder()
-        .name(nameMapping.getColumnName(table).get(columnMetadata))
+        .name(nameMapping.getColumnNames(table).get(columnMetadata))
         .type((GraphQLOutputType) getGraphQLType(columnMetadata.type(), false));
   }
 
+  private Set<GraphQLType> buildTypesForUdt(UserDefinedType udt) {
+
+    GraphQLObjectType.Builder outputType =
+        GraphQLObjectType.newObject().name(nameMapping.getUdtNames().get(udt));
+    for (Column columnMetadata : udt.columns()) {
+      try {
+        outputType.field(
+            new GraphQLFieldDefinition.Builder()
+                .name(nameMapping.getFieldNames(udt).get(columnMetadata))
+                .type((GraphQLOutputType) getGraphQLType(columnMetadata.type(), false))
+                .build());
+      } catch (Exception e) {
+        log.error(String.format("Type for %s could not be created", columnMetadata.name()), e);
```

Comment:
Again following existing patterns, but I think it's not the best way to surface errors, see #256.